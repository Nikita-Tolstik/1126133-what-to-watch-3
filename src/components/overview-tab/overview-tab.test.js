import React from 'react';
import renderer from 'react-test-renderer';
import OverviewTab from './overview-tab.jsx';

const promoMock = {
  id: 1,
  title: `Fantastic Beasts: The Crimes of Grindelwald2`,
  posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  description: `Fantastic Beasts: The Crimes of Grindelwald`,
  rating: 5,
  scoresCount: 7,
  director: `Wes Andreson`,
  stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
  runTime: 55,
  genre: `Action`,
  released: 44,
  isFavorite: true,
  videoLink: `https://upload.wikimedia.org/`,
  videoPreview: `https://upload.wikimedia.org/`,
};

it(`Render OverviewTab`, () => {
  const tree = renderer.create(
      <OverviewTab
        film={promoMock}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
