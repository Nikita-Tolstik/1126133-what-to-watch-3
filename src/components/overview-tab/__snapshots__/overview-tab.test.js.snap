// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render OverviewTab 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      5
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        7
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    Fantastic Beasts: The Crimes of Grindelwald
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Wes Andreson
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Bill Murray, Edward Norton, Jude Law
      </strong>
    </p>
  </div>,
]
`;
